pkgs/by-name/ca/cato/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  stdenv,
+  fetchurl,
+  dpkg,
+  lib,
+  libz,
+  autoPatchelfHook,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cato";
+  version = "5.2.1.1";
+
+  src = fetchurl {
+    url = "https://clients.catonetworks.com/linux/${version}/cato-client-install.deb";
+    sha256 = "sha256-0hUchaxaiKJth2ByQMFfjsCLi/4kl+SrNSQ33Y6r3WA=";
+  };
+
+  dontConfigure = true;
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    dpkg
+  ];
+
+  buildInputs = [
+    libz
+    stdenv.cc.cc
+  ];
+
+  unpackPhase = ''
+    runHook preUnpack
+    dpkg -x $src source
+    cd source
+    runHook postUnpack
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    mkdir $out
+
+    mv usr/lib $out/lib
+    mv lib/systemd $out/lib/systemd
+    substituteInPlace "$out/lib/systemd/system/cato-client.service" --replace /usr/sbin/ "$out/bin/"
+
+    mkdir -p $out/bin
+    mv usr/sbin/* $out/bin
+    mv usr/bin/* $out/bin
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Cato";
+    homepage = "https://www.catonetworks.com/";
+    license = licenses.unfree;
+    maintainers = with maintainers; [ yarekt ];
+    platforms = [ platforms.linux ];
```

Comment:
Since this is a prebuilt package (https://github.com/NixOS/nixpkgs/pull/339533#discussion_r1746808083), I assume it is build for `x86_64-linux`? In this case, `platforms` should not be set to _all_ Linux platforms.